client/my-sites/upgrade-nudge/expanded/index.jsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import page from 'page';
+
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import QueryPlans from 'components/data/query-plans';
+import Gridicon from 'components/gridicon';
+import PlanCompareCard from 'my-sites/plan-compare-card';
+import PlanCompareCardItem from 'my-sites/plan-compare-card/item';
+import TrackComponentView from 'lib/analytics/track-component-view';
+import formatCurrency from 'lib/format-currency';
+import { preventWidows } from 'lib/formatting';
+import { getFeatureTitle } from 'lib/plans';
+import { getPlanBySlug } from 'state/plans/selectors';
+import { PLAN_PERSONAL, getPlanClass, plansList } from 'lib/plans/constants';
+import analytics from 'lib/analytics';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+const ExpandedUpgradeNudge = ( {
+	translate,
+	plan = {},
+	currentPlan = {},
+	planConstants = {},
+	planClass,
+	features,
+	upgrade,
+	benefits,
+	title,
+	subtitle,
+	highlightedFeature,
+	eventName = 'calypso_upgrade_nudge_impression',
+	event='',
+	children,
+	siteSlug
+} ) => {
+	//Display only if upgrade path available
+	if (
+		! currentPlan ||
+		( planConstants.availableFor && ! planConstants.availableFor( currentPlan.productSlug ) )
+	) {
+		return null;
+	}
+
+	const price = formatCurrency( plan.raw_price / 12, plan.currency_code );
+	const eventProperties = {
+		cta_size: 'expanded',
+		cta_name: event,
+		cta_feature: highlightedFeature
+	};
+
+	if ( ! features ) {
+		if ( planConstants.getPromotedFeatures ) {
+			features = planConstants.getPromotedFeatures().filter( feature => feature !== highlightedFeature ).slice( 0, 6 );
+		} else {
+			features = [];
+		}
+	}
+
+	return (
+		<Card className="upgrade-nudge-expanded">
+			<QueryPlans />
+			<TrackComponentView { ...( { eventName, eventProperties } ) } />
+			<div className="upgrade-nudge-expanded__business-plan-card">
+				<PlanCompareCard
+					title={ plan.product_name_short }
+					line={ translate( '%(price)s per month, billed yearly', { args: { price } } ) }
+					buttonName={ translate( 'Upgrade' ) }
+					onClick={ () => {
+						analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', eventProperties );
```

Comment:
Let's use the `recordTracksEvent` from `state/analytics/actions`. Should we ever want to test this component it will be a lot easier to do so. ( The analytics lib assumes that window is available. )